supplier_server_mock/supplier_server/app.py

```diff
@@ -142,7 +142,7 @@ def booking():
     if not reservation_id:
         raise exceptions.BadRequest(1000, 'Missing argument', 'Required argument \'reservation_id\' was not found')
     try:
-        expires_at, variant_quantity_map, product_id = utils.decode_reservation_data(reservation_id)
+        expires_at, variant_quantity_map, product_id, date = utils.decode_reservation_data(reservation_id)
```

Comment:
`date` is already used for `from datetime import date`. Use eg. `booking_date_time`.